pkg/dispatcher/controller/kafkachannel_test.go

```diff
@@ -177,6 +178,10 @@ func NewMockDispatcher(t *testing.T) MockDispatcher {
 func (m MockDispatcher) Shutdown() {
 }
 
-func (m MockDispatcher) UpdateSubscriptions(_ []eventingduck.SubscriberSpec) map[eventingduck.SubscriberSpec]error {
+func (m MockDispatcher) UpdateSubscriptions(subscriberSpecs []eventingduck.SubscriberSpec) map[eventingduck.SubscriberSpec]error {
+	return nil
+}
```

Comment:
you added back in the variable name instead of _ which will cause intellij to flag it as unused?  any reason - still appears not to be used?